packages/edit-widgets/src/store/actions.js

```diff
@@ -72,5 +75,13 @@ export function* saveWidgetAreas() {
 				content: serialize( blocks ),
 			}
 		);
+		yield dispatch(
+			'core/notices',
+			'createSuccessNotice',
+			__( 'Block areas saved succesfully.' ),
+			{
+				id: WIDGET_AREAS_SAVE_NOTICE_ID,
+				type: 'snackbar',
```

Comment:
I wondered whether the "context" argument should be used here to differentiate between the "post editor" notices and the "widget screen" notices. Imagine a world where the WP-Admin is a full SPA, do we want to use this context to differentiate between these notices?

Also depending on the answer, this raises the question of whether we should set a default context to "post editor" for existing plugins creating notices in the editor.

Reply:
Hi @youknowriad, you mean the context for translation right? If yes even if we a are in a full SPA I don't think we need context to differentiate between screens. WordPress did not used different contexts if two strings were on different screens so I think we can follow the same logic.

Reply:
no, I meant the "context" property of the notices (we never used it but the API supports it). cc @aduth as we discussed some of this recently.